nailgun/entities.py

```diff
@@ -2210,6 +2212,9 @@ def path(self, which=None):
     def add_puppetclass(self, synchronous=True, **kwargs):
         """Add a Puppet class to host
 
+        Here is an example of how to use this method::
+            hostgroup.add_puppetclass(data={'puppetclass_id': puppet.id})
+
```

Comment:
Since this method is very specific to adding a puppet class, shouldn't we make it easier to use and instead of having a `**kwargs` argument, use a `class_id` argument? Then the method can pass `data={'puppetclass_id': class_id}` explicitly and using this method would be simpler: `hostgroup.add_puppetclass(my_puppet_class.id)`.